railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -55,8 +55,10 @@ group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
 
+<%- unless options.skip_bundler_audit? -%>
```

Comment:
Can you move this above the newline above so that newline is only included if `bundler-audit` is included?